File: bots/turkish/qxwsc.cpp
// C++ ile yazılmıştır.

#include <iostream>
#include <tl-cpputils/telegram.h>

int main() {
    // Token ile botu oluşturun ve çalıştırın
    tl::TelegramBot bot("TOKEN");
    bot.start();

    // /start komutunu kullanarak botun adını gönderin
    bot.command("start", [&](tl::Update update) {
        tl::Api api(bot);
        api.sendMessage(update.message.chat.id, "Merhaba benim adım Qiyas");
    });

    // /link komutunu kullanarak linkli buton gönderin
    bot.command("link", [&](tl::Update update) {
        tl::Api api(bot);
        api.sendMessage(update.message.chat.id, "Butona tıklayın", {}, {
            {
                "type": "button",
                "text": "Tıkla",
                "url": "https://t.me/qiyascc"
            }
        });
    });

    // Botu çalıştırın
    bot.run();
    return 0;
}

File: bots/turkish/qxwsc.js
const Telegraf = require('telegraf');

const bot = new Telegraf('TOKEN');

// Bu fonksiyon /start komutunu kullanarak botun adını gönderir
bot.command('start', (ctx) => ctx.reply('Merhaba benim adım qxwsc'));

// Bu fonksiyon /link komutunu kullanarak linkli buton gönderir
bot.command('link', (ctx) => ctx.reply('Butona tıklayın', Telegraf.Markup
    .inlineKeyboard([
        Telegraf.Markup.urlButton('Tıkla', 'https://t.me/qiyascc')
    ]).extra()
));

bot.launch();
